recipe/bld.bat

```diff
@@ -24,15 +24,22 @@ set ^"MESON_OPTIONS=^
  ^"
 
 :: configure build using meson
-meson setup builddir !MESON_OPTIONS!
+meson setup forgebuild !MESON_OPTIONS!
 if errorlevel 1 exit 1
 
 :: print results of build configuration
-meson configure builddir
+meson configure forgebuild
 if errorlevel 1 exit 1
 
-ninja -v -C builddir -j %CPU_COUNT%
+ninja -v -C forgebuild -j %CPU_COUNT%
 if errorlevel 1 exit 1
 
-ninja -C builddir install -j %CPU_COUNT%
+ninja -C forgebuild install -j %CPU_COUNT%
+if errorlevel 1 exit 1
+
+:: create directory for modules so post-link script doesn't fail
+set "MODULEDIR=%LIBRARY_LIB%\gtk-3.0\3.0.0"
```

Comment:
Why is this necessary now? I thought this folder was created during `ninja install`?

Reply:
Well, when I was trying out the error catching, I found that this has been silently erroring the whole time. That directory is not created during the install because the modules are built statically on Windows (a default upstream choice). So while nothing was broken with the built in modules because of this, it was still erroring. That error apparently wasn't caught because it was in the shell redirection and not in the program itself, so the errorlevel was not set. With this in there, the (empty) directory is now a part of the package as the tools would expect